tests/test_model_framework.py

```diff
@@ -53,15 +53,21 @@ class TestModelFrameworkMST(SMRFTestCase):
     """
     Test timezone handling for MST.
     """
-    TIMEZONE = pytz.timezone('US/Mountain')
```

Comment:
Little did I know that MST is the way to go and 'US/Mountain' is deprecated

Reply:
`US/Mountain` will also apply daylight savings time so you can either miss an hour or double up.